nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -34,6 +34,7 @@
 from omegaconf import DictConfig
 
 from nemo.collections.asr.modules import rnnt_abstract
+from nemo.collections.asr.modules.hat import HATJoint
```

Comment:
No modules should be imported inside of Greedy of Beam decoding libraries because it will eventually cause circular dependency